sql/synth_recipes.sql

```diff
@@ -4750,6 +4750,7 @@ INSERT INTO `synth_recipes` VALUES (75528,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
 INSERT INTO `synth_recipes` VALUES (75529,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4157,0,0,2779,2779,2779,2779,1,1,1,1,'Enfeeb. Kit Poison');
 INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4163,0,0,2780,2780,2780,2780,1,1,1,1,'Enfeeb. Kit Blind');
 INSERT INTO `synth_recipes` VALUES (75531,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4161,0,0,2781,2781,2781,2781,1,1,1,1,'Enfeeb. Kit Sleep');
+INSERT INTO `synth_recipes` VALUES (75532,1,0,0,0,0,8,34,0,0,0,4100,4242,12994,0,0,0,0,0,0,0,818,818,848,848,2,3,1,2,'Shoes (desynth)');
```

Comment:
To date, no desynth has been proven to use the sub-skills. Whiel you cited no sources, I looked them up my self and no source says this. Ffxiclopedia lists only the 34 leather, bg doesn't list a desynth at all (not all synth-able items are desynth-able, but BG misses a lot of ones that are) and jp wiki's don't mention either making out only source 'clopedia.

Reply:
Thank you for this information, I will fix that

Reply:
IDs 19941, 33530, 39933, 49956, 49968, 50502, 75517, 75518 are desynths with a sub too.
Want me to fix them at the same time ?

Reply:
yes

Reply:
done